caas/kubernetes/provider/k8s.go

```diff
@@ -55,7 +55,13 @@ import (
 	"github.com/juju/juju/storage"
 )
 
-var logger = loggo.GetLogger("juju.kubernetes.provider")
+var (
+	logger         = loggo.GetLogger("juju.kubernetes.provider")
+	jujudStartUpSh = `
```

Comment:
this should be in a standard place

Reply:
should we put it to `caas/kubernetes/provider/bootstrap.go` ?

Reply:
somewhere in juju/juju/caas as it is not k8s specific